adapters/bridge.go

```diff
@@ -12,6 +13,10 @@ import (
 	"github.com/smartcontractkit/chainlink/store/models"
 )
 
+var (
+	BridgeResultMustBeJSONObjectError = errors.New("Bridge result must be a valid JSON object")
```

Comment:
I believe the convention for errors is to have the prefix `Err` or `Error`. I understand we didn't have that before with the private variable.